index.md

```diff
@@ -32,27 +32,27 @@ We've created a full [Getting Started guide](https://docs.google.com/document/d/
 Focus area | Designer
 --- | ---
 Admin perspective | [Brie Mignano](https://github.com/bmignano)
+Admin perspective | [Megan Hall](https://github.com/megan-hall)
+Admin perspective & OCM | [Gina Doyle](https://github.com/gdoyle1)
 Admin perspective & OLM | [Colleen Hart](https://github.com/beanh66)
-Che / CodeReady workspaces | [Beau Morley](https://github.com/beaumorley)
+Dev perspective | [Laura Wright](https://github.com/lwrigh)
+Dev perspective, Che / CodeReady workspaces | [Beau Morley](https://github.com/beaumorley)
 Dev perspective, Add flows / Dev Catalog | [Parvathy Vijayakumary Ramachandran](https://github.com/parvathyvr)
```

Comment:
@megan-hall Can we change Parvathy to the following:

Dev perspective, Che / CodeReady workspaces | [Parvathy Vijayakumary Ramachandran](https://github.com/parvathyvr)